modules/web/src/app/shared/entity/node.ts

```diff
@@ -161,9 +161,16 @@ export class NodeSystemInfo {
 export class AnexiaNodeSpec {
   vlanID: string;
   templateID: string;
+  template: string;
   cpus: number;
   memory: number;
   diskSize: number;
```

Comment:
Since we're not using `diskSize` anymore so should we delete this field as well?

Reply:
The `diskSize` attribute is technically still available in the machine-controller, but deprecated. I would expect both specs to be similar.